src/pkg/exporter/cache.go

```diff
@@ -0,0 +1,88 @@
+package exporter
+
+import (
+	"sync"
+	"time"
+)
+
+var c *cache
+
+type cachedValue struct {
+	Value      interface{}
+	Expiration int64
+}
+
+type cache struct {
+	CacheTime int64
+	store     map[string]cachedValue
+	*sync.RWMutex
+}
+
+// CacheGet get a value from cache
+func CacheGet(key string) (value interface{}, ok bool) {
```

Comment:
If this func is not intended to be called outside of the package, you should not export it.

The exporter should be pretty self contained, i.e. the pkg should expose relative minimum funcs.

Please go through all other funcs and objects, for example, it seems you don't need to export HarborClient either.

Reply:
actually, cache can be independent pkg but exporter case is simple. so all the component shared the same package name